AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-details/booking-details.component.ts

```diff
@@ -104,7 +104,7 @@ export class BookingDetailsComponent implements OnInit, OnDestroy {
 
     setTimeObserver() {
         if (this.booking) {
-            const endofday = this.booking.scheduled_date_time;
+            const endofday = new Date(this.booking.scheduled_date_time);
```

Comment:
Was this change put in because `this.booking.scheduled_date_time` wasn't always a Date? Might be worth checking it's truthy before doing this, if there's a chance it can be undefined etc

Reply:
No this was because the scheduled_date_time was being linked to the endofday const rather than just taking the value so anytime it was updated the scheduled_date_time would be updated which is not what we want. The new Date solved this